app/models/apple/config.rb

```diff
@@ -24,40 +20,6 @@ class Config < ApplicationRecord
     delegate :public_feed, to: :podcast, allow_nil: true
     alias_method :private_feed, :feed
 
-    def self.find_or_build_private_feed(podcast)
```

Comment:
not really needed in this particular PR but moved to the new subclass. looking at it now, wondering if it's even needed now? seems like its usage was to create a feed specifically for an `apple_config` and now the subclass serves that purpose, i think.

Reply:
Yeah, I think some of that can leave now.  Will comment below.